src/bin/tikv-importer.rs

```diff
@@ -107,6 +107,13 @@ fn main() {
     util::print_tikv_info();
     check_environment_variables();
 
+    if tikv_util::panic_mark_file_exists(&config.storage.data_dir) {
```

Comment:
Can this be extracted since there is one in tikv-server as well?

Reply:
Actually panic_mark_file_exists is enough, it is simple and clear.